plugin.rb

```diff
@@ -144,12 +170,21 @@ class Engine < ::Rails::Engine
   end
 
   add_to_serializer(:post, :current_user_used_main_reaction) do
-    return false unless scope.user.present?
+    return false if scope.is_anonymous?
```

Comment:
you know how much I prefer this :hugs:

Reply:
I ruthlessly prune `unless` everywhere I go 😆